Archipelago.cpp

```diff
@@ -20,6 +20,7 @@
 
 #define AP_OFFLINE_SLOT 1404
 #define AP_OFFLINE_NAME "You"
```

Comment:
Should have been `John Doe` :)

Reply:
This way, if there is a message and you construct for example, "`target` got item", you get " You got item" which i thought was funny :3